src/app/pages/learning-research/research.scss

```diff
@@ -0,0 +1,106 @@
+@import 'pattern-library/core/pattern-library/headers';
+
+@include wider-than($phone-max) {
+    .mobile-only {
+        display: none;
+    }
+
+    .ReactModal__Content {
```

Comment:
I have some concerns that after loading, these `!important` specs will override values for other pages, since they're not scoped to a particular className here.